RELEASE-NOTES.md

```diff
@@ -19,6 +19,13 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 
 ## [Unreleased] - Release date yyyy-mm-dd
 
+### Changed
+- Upgrades `vcpkg` usage for axom's automated Windows builds to its
+  [2024.03.19 release](https://github.com/microsoft/vcpkg/releases/tag/2024.03.19).
+  Also updates vcpkg port versions for axom dependencies. Temporarily removes `umpire`
+  from axom's default dependencies on Windows due to incompatibility between umpire's
+  external `fmt` and axom's vendored copy.
```

Comment:
This is the gift that keeps on giving.. is there a path forward?

Reply:
Probably best to excise the Axom internal copy of fmt and move to external?

Reply:
I think axom will need to support an external fmt, and this should be the default for uberenv builds.

For basic testing of axom, I think we'll want to also support an internal vendored fmt when an external one is not provided. E.g. we have some azure CI tests for windows and mac that build w/o any TPLs.

We should probably coordinate how we support this w/ Umpire and Conduit (and other RADIUSS fmt users/providers), e.g. to support header-only fmt vs. compiled fmt, and possibly other flags.

Tag: @davidbeckingsale @cyrush

Reply:
I created a ticket to track this: https://github.com/LLNL/axom/issues/1302